tutor/specs/integration/assignment-edit.spec.js

```diff
@@ -249,15 +249,16 @@ context('Assignment Edit', () => {
     })
   });
 
-  it('shows error if due date is before open date', () => {
+  it.only('shows error if due date is before open date', () => {
     typedDueDate = moment().subtract(1, 'weeks').format('MMM D [| 05:00 PM]')
```

Comment:
I think you probably left the `.only` on by mistake?

Reply:
yes :) thanks for catching it.